### DISCUSSION TITLE
What is considered a harvester event?

### DISCUSSION THREAD

**Discussion by cipriani14** [martomi/chiadog#61] [Q&A]
I’ve been receiving alerts every 5 minutes for the last few hours that my harvester may be down because there have been no events for XXXX seconds. However, I’m watching the CLI plots progress through. 

What is it looking for that it’s not seeing?

**Comment by martomi** [comment]
These are harvester events: https://github.com/martomi/chiadog/blob/main/tests/chia_log/logs/harvester_activity/nominal.txt

If it's not seeing that, likely you have not configured the correct path to the logs or haven't enabled INFO logging level on Chia. Please double-check that and the README.

**Comment by cipriani14** [comment]
Interesting. It’s definitely seeing the logs, because it works for awhile, and then stops seeing the logs. I just launched it and will let it run for a bit.. it sees everything, including the right number of plots I have. I am on Windows, btw.

I wonder if this has to do with the log splitting up into different files after they get to a certain size? For example, I have a main debug.log file that’s slowly growing in size as the log writes to it. But then I have 7 more debug.log.x files, where x is a number from 1-7.

This morning I restarted the script and it stopped working with an hour, however the day before it worked for around 16 hours before I started getting a message about my harvester potentially being down.

**Reply by martomi** [reply]
I believe there's a known issue on Windows that it stops working after the log rotates. Will let you know once that's resolved.

[a comment by an automated account is not shown]
